annotation/tests/test_annotations.py

```diff
@@ -0,0 +1,130 @@
+import uuid
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+
+from annotation.annotations.main import row_to_dict
+from annotation.models import AnnotatedDoc, Category, File, Job, User
+from annotation.schemas.categories import CategoryTypeSchema
+from annotation.schemas.jobs import JobTypeEnumSchema, ValidationSchema
+from tests.override_app_dependency import TEST_TENANT
+
+
+@pytest.fixture(scope="function")
```

Comment:
Default scope of fixture is function, you can omit this. The same for other fixtures